beanstalkworker_demo/job1.go

```diff
@@ -1,6 +1,6 @@
 package main
 
-import "github.com/tomponline/beanstalkworker/beanstalkworker"
+import "beanstalkworker/beanstalkworker"
```

Comment:
@PlkMarudny will this relative import still work with go get and vendoring tools?

Reply:
Seems to be working with "go get" and when vendored (using "dep")

Reply:
@PlkMarudny i think id prefer to keep this using the absolute path, so that others can try it out from their own repos and pull in the external depedency.